packages/back-end/src/routers/sdk-connection/sdk-connection.controller.ts

```diff
@@ -191,6 +191,45 @@ export const checkSDKConnectionProxyStatus = async (
   });
 };
 
+export const getSDKConnectionsWebhooks = async (
+  req: AuthRequest,
+  res: Response<{
+    status: 200;
+    webhooks: Record<string, WebhookInterface[]>;
+  }>
+) => {
+  const context = getContextFromReq(req);
+
+  // Get all connections to check permissions
+  const connections = await findSDKConnectionsByOrganization(context);
+
+  // Fetch webhooks for all connections
+  const webhooksByConnection: Record<string, WebhookInterface[]> = {};
+
+  await Promise.all(
+    connections.map(async (connection) => {
+      const webhooks = await findAllSdkWebhooksByConnection(
+        context,
+        connection.id
+      );
+
+      // If user does not have write access, remove the shared secret
+      if (!context.permissions.canUpdateSDKWebhook(connection)) {
+        webhooks.forEach((w) => {
+          w.signingKey = "";
+        });
```

Comment:
Not a big deal, but do we ever need this field in this context?  Feels like we should just return the minimum set of fields needed in this context.  So just id, name, and error.  Can make a new `SdkWebhookSummaryInfo` type

Reply:
added a summary type (with a few other summary fields, just in case)